lib/tasks/static_pages.rake

```diff
@@ -20,7 +20,7 @@ namespace :static_pages do
       outpath = File.join([Rails.root, 'public', output])
       resp = app.get(route)
       if resp == 200
-        File.delete(outpath) unless not File.exist?(outpath)
+        File.delete outpath if File.exist?(outpath)
```

Comment:
John, where Mark had had had had had had had. Had had had had a better effect on the teacher.